app/src/main/java/com/example/yp_playlist/presentation/media/MediaViewModel.kt

```diff
@@ -39,22 +30,8 @@ class MediaViewModel(
             it.trackId == trackId
         }
         if (track != null) {
```

Comment:
🍏 Можно заменить на использование одной из scope функций языка Kotlin:
```suggestion
        track?.let { _trackInfo.value =  it }
```
Больше о scope functions: https://kotlinlang.org/docs/scope-functions.html